services/views/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "pins-views",
+  "version": "1.0.0",
+  "description": "nunjucks views used by pins",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
+    "format": "prettier --write './**/*.js'",
+    "test": "jest --coverage --maxWorkers=4"
+  },
+  "dependencies": {
+    "govuk-frontend": "^3.13.0",
```

Comment:
It doesn't look like govuk-frontend is being used in this module but is being called from the app consuming the module, so perhaps we don't need to include it here?

Reply:
I was wondering. Removed.